mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -61,7 +58,8 @@ def run_create_projects():
     project_type_classes = {
         1: BuildAreaProject,
         2: FootprintProject,
-        3: ChangeDetectionProject,
+        3: BuildAreaProject,
```

Comment:
no this isn't a mistype, maybe it would be better if @Hagellach37  answers this modification. Basically because `Completeness` and `ChangeDetection` both have almost the same structure. Both projects require two urls.